src/main/java/uk/gov/companieshouse/search/api/service/search/impl/alphabetical/AlphabeticalSearchRequestService.java

```diff
@@ -230,7 +232,7 @@ private Company getCompany(SearchHit hit) {
         company.setOrderedAlphaKeyWithId((String) sourceAsMap.get(ORDERED_ALPHA_KEY_WITH_ID));
         company.setKind(ALPHABETICAL_KIND);
 
-        companyLinks.setSelf((String) (links.get("self")));
+        companyLinks.setCompanyProfile((String) (links.get("self")));
         company.setLinks(companyLinks);
 
         company.setCompanyType((String) sourceAsMap.get("company_type"));
```

Comment:
We may want to consider refactoring this to be in the mapper class as it's performing a mapping function. Not required for this PR though.